### ISSUE TITLE
Addon-docs: Docs mode shouldn't render "Canvas" & "Docs" tabs

### ISSUE THREAD

**Issue by patricklafrance** [storybookjs/storybook#9518]
**Describe the bug**

In 5.2 when Storybook is compile in `--docs` mode the "Canvas" & "Docs" tabs are not rendered, which is great.

In 5.3.6 it doesn't seems to be the case anymore:

![image](https://user-images.githubusercontent.com/794579/72642384-321eff00-393a-11ea-82a9-052a2378b257.png)

**To Reproduce**
1. Start storybook with `--docs`

**Expected behavior**

"Canvas" & "Docs" tabs are not rendered.

![image](https://user-images.githubusercontent.com/794579/72642430-5a0e6280-393a-11ea-8ccc-dd667cf0ed76.png)

**Screenshots**

**Code snippets**
If applicable, add code samples to help explain your problem.

**System:**

Environment Info:

  System:
    OS: Windows 10 10.0.17763
    CPU: (8) x64 Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
  Binaries:
    Node: 12.5.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.21.1 - C:\Dev\20_gsoft\sg-orbit\node_modules\.bin\yarn.CMD
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.17763.831.0

**Additional context**

**Comment by shilman** [comment]
@patricklafrance Unable to repro. Can you try removing node_modules, reinstalling, maybe `localStorage.clear()`, removing `.cache`? I think it's something funny on your machine. (NOTE: it's messed up that `--docs` is sensitive to this stuff, but I really don't know what's going on)

**Comment by patricklafrance** [comment]
Ok thanks for the reply, i’ll do some testing and keep you posted.

**Comment by patricklafrance** [comment]
@shilman the issue occurs when I specify one of these:

```
addParameters({
    options: {
        showPanel: false,
        panelPosition: "right",
});
```

**Comment by tscolbert** [comment]
I'm having this same issue. I removed node_modules, cleared localStorage, and I don't have a .cache to remove. Is there anything else I can do to get the --docs mode to work?

**Comment by shilman** [comment]
Are you using the `options` global parameter like @patricklafrance mentioned above?

**Comment by Zunaib** [comment]
@patricklafrance is right, works without the
```options```.
But if you don't want canvas at all, why would you want a panel position ?
Anyways i think panel position has a preference over the --docs tag.

**Comment by tscolbert** [comment]
@Zunaib changing the options doesn't make any sort of difference for me. 

Right now my config is only

```
import {configure} from '@storybook/react';
import "./stories.scss";

configure(require.context('../src/components', true, /\.stories\.tsx$/), module);
```

And I'm still having the issue, after again repeating what @shilman suggested.

Could it be an issue related to to using typescript w/ react?

**Comment by Zunaib** [comment]
Can you show me you files in .storybook/ folder and the storybook version. 
Might suggest a fix

**Comment by patricklafrance** [comment]
@Zunaib

This is because I deploy 2 storybook, one with Stories & one with Docs only. They both share the same config.

I fix it by not setting the options when I compile my SB for docs only, but I don't see why it wouldn't be fine out of the box?

```
let optionsMessingWithDocsMode = {};

if (!isDocs) {
    optionsMessingWithDocsMode = {
        showPanel: false,
        panelPosition: "right"
    };
}

addParameters({
    options: {
        ...optionsMessingWithDocsMode,
        theme: customStorybookTheme,
        storySort: customStorySort
    },
    docs: {
        inlineStories: true
    }
});
```

**Comment by Zunaib** [comment]
This should be looked into then by the Storybook Team. I'd try tr reproduce it and get back.

**Comment by Zunaib** [comment]
How's your file structure, is it the new one. Have you migrated yet to new files ?

**Comment by danielbayerlein** [comment]
Exists a solution for this problem?

**Comment by prsdta** [comment]
Hi, passing by as I had the same issue, @shilman advice worked for me.

I just had to clean the `sessionStorage` since I used the same browser to browse the development version and the `--docs` version.

**Comment by shilman** [comment]
This is fixed in 7.0 https://storybook.js.org/blog/storybook-7-docs/
